app/models/issue_template.rb

```diff
@@ -45,4 +45,29 @@ def check_default
       IssueTemplate.search_by_project(project_id).search_by_tracker(tracker_id).update_all(is_default: false)
     end
   end
+
+  #
+  # Class method
+  #
+  class << self
+    def get_inherit_templates(project_ids, tracker_id)
+      # keep ordering of project tree
+      inherit_templates = []
+      project_ids.each do |i|
+        inherit_templates.concat(IssueTemplate.search_by_project(i)
+                                     .search_by_tracker(tracker_id)
+                                     .enabled
+                                     .enabled_sharing
+                                     .order_by_position)
+      end
+      inherit_templates
+    end
+
+    def get_templates_for_project_tracker(project_id, tracker_id=nil)
```

Comment:
__[rubocop]__
* Surrounding space missing in default value assignment. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-around-equals)